Backend/index.js

```diff
@@ -2,58 +2,80 @@ const { PrismaClient } = require("@prisma/client");
+//   const { name, email, phoneNumber, password } = req.body;
+//   console.log(name);
+//   bcrypt.hash(password, saltRounds, async function (err, hashed) {
+//     try {
+//       const newUser = await prisma.User.create({
+//         data: {
+//           name,
+//           email,
+//           phoneNumber: parseInt(phoneNumber),
+//           hashedPassword: hashed,
+//         },
+//       });
+//       res.status(200).json(newUser);
+//     } catch (e) {
+//       res.status(500).json({ error: e.message });
+//     }
+//   });
+// });
+
 app.post("/signup", async (req, res) => {
-    console.log("hurray");
-    const {name, email, phoneNumber, password} = req.body;
-    console.log(name);
-    bcrypt.hash(password, saltRounds, async function(err, hashed){
-        try {
-            await prisma.User.create({
-                data:{
-                    name,
-                    email,
-                    phoneNumber: parseInt(phoneNumber),
-                    hashedPassword: hashed,
-                }
-            })
-            res.status(200).json({});
-        } catch (e) {
-            res.status(500).json({"error": e.message});
-        }
-    })
-})
+  console.log("hurray");
+  const { name, email, phoneNumber, password } = req.body;
+  console.log(name, email, phoneNumber);
+  try {
+    const hashedPassword = await bcrypt.hash(password, saltRounds);
+    const newUser = await prisma.User.create({
+        data: {
+            name,
+            email,
+            phoneNumber: parseInt(phoneNumber),
+            hashedPassword,
+        },
+    });
+    res.status(200).json(newUser);
+  } catch (e) {
+    console.log(e.message)
+    res.status(500).json({ error: e.message });
+  }
+});
+
 
 app.post("/login", async (req, res) => {
-    const { email, password } = req.body;
-    try {
-      const userRecord = await prisma.user.findUnique({ where: { email } });
-      if (!userRecord) {
-        return res.status(404).json({ error: "User not found" });
-      }
-      const isMatch = await bcrypt.compare(password, userRecord.hashedPassword);
-      if (isMatch) {
-        res.status(200).json({});
-      } else {
-        res.status(401).json({ error: "Invalid credentials" });
-      }
-    } catch (e) {
-      res.status(500).json({ error: e.message });
+  const { email, password } = req.body;
+  try {
+    const userRecord = await prisma.user.findUnique({ where: { email } });
+    if (!userRecord) {
+      return res.status(404).json({ error: "User not found" });
     }
+    const isMatch = await bcrypt.compare(password, userRecord.hashedPassword);
+    if (isMatch) {
+      res.status(200).json({});
+    } else {
+      res.status(401).json({ error: "Invalid credentials" });
+    }
+  } catch (e) {
+    res.status(500).json({ error: e.message });
+  }
 });
 
-app.get('/homepage', (req, res) => {
-    res.send("welcome to app")
- })
+app.get("/homepage", (req, res) => {
+  res.send("welcome to app");
+});
```

Comment:
I believe there are some packages that can help with auto-checking and formatting, such as single/double quotes, colons, indenting, etc. Can you explore one of them and try to set it up?